lib/ibmdb.js

```diff
@@ -399,6 +414,7 @@ IBMDB.prototype.buildInsertDefaultValues = function(model) {
  */
 IBMDB.prototype.updateOrCreate = IBMDB.prototype.save =
   function(model, data, options, callback) {
+    debug('IBMDB.prototype.updateOrCreate (enter)');
```

Comment:
Probably you can add more useful info to the debug like what `model`, `data`, `option`are passing? Thought?